shared/src/taxClearanceCertificate.ts

```diff
@@ -2,6 +2,18 @@ import { orderBy } from "lodash";
 import taxClearanceCertificateAgenciesJSON from "../../content/src/mappings/taxClearanceCertificateIssuingAgencies.json";
 import { StateObject } from "./states";
 
+export type TaxClearanceCertificateResponse = {
+  error?: {
+    type:
+      | "INELIGIBLE_TAX_CLEARANCE_FORM"
+      | "FAILED_TAX_ID_AND_PIN_VALIDATION"
+      | "MISSING_FIELD"
+      | "SYSTEM_ERROR";
```

Comment:
The API we’re integrating with returns a 400 status code along with a specific error message when a failure occurs in one of their internal systems (referred to as a Natural Program Error). Even though this comes back as a 400, we’re treating it as a system-level error on our end, and mapping it to a standardized SYSTEM_ERROR type for consistency.

This is distinct from other internal server errors that may occur, which will still throw and be handled as actual 500-level errors. The goal is to provide a consistent response shape while still recognizing and logging true unhandled exceptions appropriately.